src/server/control.rs

```diff
@@ -54,7 +56,38 @@ pub fn instance_command(cmd: &InstanceCommand) -> anyhow::Result<()> {
     };
     let os = detect::current_os()?;
     let methods = os.get_available_methods()?.instantiate_all(&*os, true)?;
-    let inst = get_instance(&methods, name)?;
+    let inst = match get_instance(&methods, name) {
```

Comment:
This original `get_instance` was used for all commands not just status. This `RemoteStatus` code is supposed to be only for statuses so have be moved to `Status` case.